WordPressAuthenticator/Extensions/URL+JetpackConnect.swift

```diff
@@ -0,0 +1,7 @@
+import Foundation
+
+extension URL {
+    public func isJetpackConnect() -> Bool {
```

Comment:
nit: maybe we can do a computed property instead of a method here?

Reply:
Done.